test/PullRequests.test.js

```diff
@@ -0,0 +1,16 @@
+const PullRequests = require('../src/PullRequests')
+
+test('.isIgnorable(pr) returns true with matched strings', () => {
+  const pullRequest = new PullRequests()
+  const pullRequests = [
+    { title: "Dont merge - this is a PR title" },
+    { title: "Don't merge - this is a PR title" },
+    { title: "Do not merge - this is a PR title" },
+    { title: "[DO NOT MERGE] - this is a PR title" },
+    { title: "WIP - this is a PR title" }
```

Comment:
Sure! I actually didn't bother to run the lint when I was implementing this so I totally missed the comma-dangle. Fixed here https://github.com/ohbarye/review-waiting-list-bot/pull/18/commits/b2a8abab23eb82e16c72c2e7699e7c592a9a38fb